README.md

```diff
@@ -58,81 +54,52 @@ Include this badge in your readme if you make a new module that implements inter
 
 A valid (read: that follows this interface) IPLD format implementation the following API.
 
-### IPLD format utils
+IPLD Format APIs are restricted to a single IPLD Node, they never access any linked IPLD Nodes.
 
-#### `util.serialize(dagNode, callback)`
 
-> serializes a dagNode of an IPLD format into its binary format
+### `serialize(IpldNode)`
 
-`callback` must have the signature `function (err, binaryBlob)`, where `err` is an Error is the function fails and `binaryBlob` is a Buffer containing the serialized version.
+> Serializes the internal representation of an IPLD Node into a binary blob.
 
-#### `util.deserialize(binaryBlob, callback)`
+`IpldNode` is a previously deserialized binary Block.
```

Comment:
this is the only remaining instance of "block", is the intent to remove the concept of "block" from this doc and just use "binary blob"?